test/Microsoft.AspNetCore.SystemWebAdapters.Framework.Tests/Microsoft.AspNetCore.SystemWebAdapters.Framework.Tests.csproj

```diff
@@ -1,7 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net48</TargetFrameworks>
+    <TargetFramework>net472</TargetFramework>
```

Comment:
why this change?

Reply:
This came in when I merged my Microsoft.AspNetCore.SystemWebAdapters.Framework.Tests.csproj with what you added. I had chose 472 just because that's what test\Microsoft.AspNetCore.SystemWebAdapters.SessionState.Framework.Tests\Microsoft.AspNetCore.SystemWebAdapters.SessionState.Framework.Tests.csproj was using and I figured they may as well match. Either version is fine.

Reply:
Yea, that's what I gathered (though I think it's arbitrary).

Reply:
I believe 4.7.2 is the recommended target because that's the version that async context/locals (and maybe some other things more modern code relies on) works more aligned with core. Not sure if that affects what we're doing though